packages/typescript-estree/src/create-program/createProjectService.ts

```diff
@@ -9,7 +9,9 @@ const createStubFileWatcher = (): ts.FileWatcher => ({
 
 export type TypeScriptProjectService = ts.server.ProjectService;
 
-export function createProjectService(): TypeScriptProjectService {
+export function createProjectService(
+  jsDocParsingMode?: ts.JSDocParsingMode,
```

Comment:
Preemptive check; IIRC this is public. Should this be some sort of options bag rather than a parameter?

Reply:
Nah this isn't public - we don't export it from either `index.ts` or `use-at-your-own-risk.ts`.